db/sql/deploy/buildings_reference/functions/suburb_locality_intersect.sql

```diff
@@ -0,0 +1,30 @@
+-- Deploy nz-buildings:buildings_reference/functions/suburb_locality_intersect to pg
+
+BEGIN;
+
+-- suburb_locality_area_of_second_largest_intersect (area of suburb with most overlap)
+    -- params: p_polygon_geometry geometry, p_excluded_id (the largest overlap)
+    -- return: double area
+
+CREATE OR REPLACE FUNCTION buildings_reference.suburb_locality_area_of_second_largest_intersect(
+    p_polygon_geometry geometry, p_excluded_id integer
+)
+RETURNS double precision AS
+$$
+
+    SELECT ST_Area(ST_Intersection(p_polygon_geometry, shape))
+    FROM buildings_reference.suburb_locality
+    WHERE shape && ST_Expand(p_polygon_geometry, 100) 
```

Comment:
We previously used ST_Expand 1000m.  Consider changing back to this or ensure all buildings are within 100m of LOCALITY/ISLAND/SUBURB/PARK_RESERVE